src/elements/content-sidebar/activity-feed/task-new/Task.scss

```diff
@@ -39,6 +39,12 @@ $bcs-task-left-offset: 10 * $bcs-task-grid + 2px;
 
 .bcs-Task-statusContainer {
     margin-top: 2 * $bcs-task-grid;
+    margin-bottom: 10px;
```

Comment:
surprising values here, would have expected the design to reference the grid (4px)

Reply:
Fixed this. Now using 4px * N.